nevergrad/functions/multiobjective/core.py

```diff
@@ -89,4 +89,34 @@ def pareto_front(self) -> List[ArgsKwargs]:
             if should_be_added:
                 new_points.append((argskwargs, losses))
         self._points = new_points
-        return [p[0] for p in self._points]
+
+    def pareto_front(self, size: Optional[int] = None, method: Optional[str] = None) -> List[ArgsKwargs]:
+        """Pareto front, as a list of args and kwargs (tuple of a tuple and a dict)
+        for the function - restricted to number elements covering as well as possible the domain 
+        (domain-covering) or the losses (loss-covering) or just randomly (random)..
+        """
+        self.filter_pareto_front()
+        if method is None:
```

Comment:
You can remove this, let's give a default value to the parameter instead

Reply:
Btw, what is "random" the best default?

Reply:
Not sure it's the best... it's the fastest...